.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Ruby
```

Comment:
This should probably be changed to something more meaningful, like `Verify links`